tasks/config.yml

```diff
@@ -51,3 +51,21 @@
   notify: restart pritunl
   tags:
     - pritunl-update
+
+- name: Check app.server_port option
+  shell: pritunl get app.server_port
+  register: app_server_port
+  failed_when: app_server_port.rc != 0
+  changed_when: False
+  when: pritunl_server_port is defined
+  tags:
+    - pritunl-reverse-proxy
+
+- name: "Set app.server_port to {{ pritunl_server_port }}"
+  shell: "pritunl set app.server_port {{ pritunl_server_port }}"
+  register: resp
+  failed_when: resp.rc != 0
+  changed_when: "'app.server_port = {{ pritunl_server_port }}' in resp.stdout"
+  when: "'app.server_port = {{ pritunl_server_port }}' not in app_server_port.stdout and pritunl_server_port is defined"
```

Comment:
In the case that you have enabled the rev proxy and wants to change the port, will this condition return false and not change to new one, right? Is that the behavior expected ?

Reply:
I reviewed the logic. **not in stdout**. :)
This CLI could return an dict to be easy to verify. =p

Reply:
Indeed